tests/ord-service/tests/api_test.go

```diff
@@ -0,0 +1,322 @@
+/*
+ * Copyright 2020 The Compass Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tests
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	directorSchema "github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/kyma-incubator/compass/tests/director/pkg/gql"
+	"github.com/kyma-incubator/compass/tests/director/pkg/idtokenprovider"
+	"github.com/kyma-incubator/compass/tests/director/pkg/ptr"
+	"github.com/kyma-incubator/compass/tests/ord-service/pkg"
+	"github.com/stretchr/testify/require"
+	"github.com/tidwall/gjson"
+)
+
+const TestApp = "mytestapp"
+
+func TestODService(t *testing.T) {
```

Comment:
Does it makes sense to add tests for some of the most common OData queries like `expand,filter etc.`. 

I think that we can trust the library that those queries will work as expected (since we have tested our custom convertions in those top-level query tests), but we should test if our implementation (particularly the JPA representation of our DB Model) matches the DB and the relations there. 

We can just create some of those more common requests to the service and expect a 200 status code without an error message. I know that those tests seem shallow but they will test the integrity between the JPA model and our DB model which is the most essential part of the ord-service.

Reply:
Definitely makes sense - tests are added.